### ISSUE TITLE
Search doesn't work

### ISSUE THREAD

**Issue by thansidwell** [CodeForPhilly/vacant-lots-proj#582]
You can't click into the search field to use it.

https://github.com/CodeForPhilly/vacant-lots-proj/assets/1965986/d1a4fae1-4fc9-45ca-a38d-c05f170d9f64

**Comment by jgrimes86** [comment]
Adding a `pointer-events` attribute to the following section of globals.css (starting on line 225) seems to fix the issue.
```
#main .maplibregl-ctrl-top-right {
  top: 20px;
  right: 20px;
  pointer-events: bounding-box;
}
```
@thansidwell is it OK if I make a pull request so you can check out this change?

**Comment by thansidwell** [comment]
Definately!

**Comment by thansidwell** [comment]
Just a thought. It seems strange a bit strange to add something to make it work even though it worked before. It seems like we should be taking something away whatever is obstructing it.

But I'm good whatever works!

**Comment by jgrimes86** [comment]
@thansidwell I think the issue is that `@import "~mapbox-gl/dist/mapbox-gl.css";` was removed from globals.css.  That css file includes a 'pointer-events: auto' attribute for all map controls, along with styling for other elements.  I assume it was removed to allow custom styling of other map features, so I'm afraid of adding it back in and breaking other things.

**Comment by thansidwell** [comment]
Ah, that makes sense now!

**Comment by thansidwell** [comment]
@jgrimes86 Also the map legend tooltip doesn't work either. I assume for a similar reason. 

https://github.com/CodeForPhilly/vacant-lots-proj/assets/1965986/5c9608cf-2c59-4b57-8252-3235f91fdefa

**Comment by jgrimes86** [comment]
Yes, that was the same issue.  I didn't know that the tooltip had been interactive.  I'm going to push a commit to my PR request branch to address that issue.

Also, I made one more small change to fix a bug with the map click function. Clicking the map had been re-directing to the last searched property.

**Comment by nlebovits** [comment]
closed by #586
